runtime/compiler/control/rossa.cpp

```diff
@@ -1654,6 +1657,24 @@ onLoadInternal(
       if (!JITServer::loadLibsslAndFindSymbols())
          return -1;
       }
+#if defined(PERSISTENT_LOGGING_SUPPORT)
+//    TODO: Get Flag from CLIENT command line.
```

Comment:
This is executed when jitserver boostraps. We need a master option that enables persistent logging at runtime. Getting the option from the client will not do.